validator/testdata/invalid_validation_message_argument.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+package federation;
+
+import "federation.proto";
+import "post.proto";
+
+option go_package = "example/federation;federation";
+
+service FederationService {
+  option (grpc.federation.service) = {
+    dependencies: []
+  };
+  rpc GetPost(GetPostRequest) returns (GetPostResponse) {};
+}
+
+message GetPostRequest {
+  string id = 1;
+}
+
+message GetPostResponse {
+  option (grpc.federation.message) = {
+    def {
+      name: "post"
+      message {
+        name: "Post"
+        args { name: "id", by: "$.id" }
+      }
+    }
+  };
+  Post post = 1 [(grpc.federation.field).by = "post"];
+}
+
+message Post {
+  option (grpc.federation.message) = {
+    def [
+      {
+        name: "res"
+        call {
+          method: "post.PostService/GetPost"
+          request { field: "id", by: "$.id" }
+        }
+      },
+      { name: "post", by: "res.post", autobind: true },
+      {
+        validation {
+          name: "invalidMessageArgument",
```

Comment:
Is there an intention behind the name lowerCamelCase ?
In the example, everything else is lower_snake_case.

Reply:
Not much! Let me fix that